Flexx/Classes/Flexx.swift

```diff
@@ -142,6 +142,16 @@ public class Flexx {
         return currentLocale
     }
     
+    /// Check locale identifier
+    /// - returns: Locale value with valid local identifier
+    public func checkLocaleId(localeId: String) -> String {
```

Comment:
It will be better to move this to a function of LocaleIDs struct, because it operates with those. Something like localeId(for locale: String)

Reply:
moved